### PR TITLE
DP-749 - Date display throughout the service

### PR DESCRIPTION
**Pull request by dpatel017**
Date display throughout the service

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersIndividualOrTrust.cshtml [hunk 1]
```diff
@@ -41,7 +42,7 @@
                             <p class="govuk-body">@details.LastName</p>
                             @if (details.DateOfBirth.HasValue)
                             {
-                                <p class="govuk-body">DoB: @details.DateOfBirth?.ToString("dd/MM/yyyy")</p>
+                                <p class="govuk-body">DoB: @details.DateOfBirth?.ToString("dd MMMM yyyy")</p>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersIndividualOrTrust.cshtml:45]: are we not able to use `ToFormattedString` here also?

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersOrganisation.cshtml [hunk 2]
```diff
@@ -224,7 +225,7 @@
                             </dt>
                             <dd class="govuk-summary-list__value">
                                 @Html.Raw(details.RegistrationDate.HasValue == true
-                                         ? $"Yes<br>On {details.RegistrationDate.Value.ToString("dd/MM/yyyy")}"
+                                         ? $"Yes<br>On {details.RegistrationDate.Value.ToString("dd MMMM yyyy")}"
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersOrganisation.cshtml:228]: Same here - why not `ToFormattedString` ?

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersIndividualOrTrust.cshtml (hunk 1 above)
Comment (dpatel017) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersIndividualOrTrust.cshtml]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersOrganisation.cshtml (hunk 2 above)
Comment (dpatel017) [reply Frontend/CO.CDP.OrganisationApp/Pages/Supplier/ConnectedEntity/ConnectedEntityCheckAnswersOrganisation.cshtml]: Done

**Review by JBaigGoaco — APPROVED** [review]

**Review by andymantell — APPROVED** [review]

**Pull request merged**